meta/autoid/autoid.go

```diff
@@ -220,6 +221,7 @@ func (alloc *allocator) NextGlobalAutoID(tableID int64) (int64, error) {
 	startTime := time.Now()
 	err := kv.RunInNewTxn(context.Background(), alloc.store, true, func(ctx context.Context, txn kv.Transaction) error {
 		var err1 error
+		txn.SetDiskFullOpt(kvrpcpb.DiskFullOpt_AllowedOnAlreadyFull)
```

Comment:
ditto as the above

Reply:
try to let all the little data change transactions to be dealt with when TiKV is disk full.
Because such scenarios are some basic ability of other processing.